lib/airbrake.js

```diff
@@ -10,6 +10,7 @@ var stackTrace = require('stack-trace');
 var hashish = require('hashish');
 var querystring = require('querystring');
 var stringify = require('json-stringify-safe');
+var execSync = require('sync-exec');
```

Comment:
@jadengore we have to support older node versions, where `child_process` doesn't have `execSync` defined.